waltid-libraries/waltid-library-commons/build.gradle.kts

```diff
@@ -0,0 +1,207 @@
+import love.forte.plugin.suspendtrans.ClassInfo
+import love.forte.plugin.suspendtrans.SuspendTransformConfiguration
+import love.forte.plugin.suspendtrans.TargetPlatform
+import love.forte.plugin.suspendtrans.gradle.SuspendTransPluginConstants
+import love.forte.plugin.suspendtrans.gradle.SuspendTransformGradleExtension
+import org.gradle.kotlin.dsl.kotlin
+import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+
+plugins {
+    kotlin("multiplatform")
+    kotlin("plugin.serialization")
+    id("maven-publish")
+    id("com.github.ben-manes.versions")
+    id("love.forte.plugin.suspend-transform") version "2.0.20-0.9.2"
+}
+
+group = "id.walt"
```

Comment:
this being multiplatform now, I think it should be `id.walt.library-commons` or similar, so all build artifacts will go under the same root group, regardless of platform

for example for the crypto lib, it's `id.walt.crypto`